app/models/payment/go_cardless.rb

```diff
@@ -18,22 +19,24 @@ def write_mandate(mandate_gc_id, scheme, next_possible_charge_date, customer_id)
 
     def write_transaction(uuid:, amount:, currency:, charge_date:, page_id:, customer_id:, payment_method_id:, subscription: nil)
       local_transaction = Payment::GoCardless::Transaction.find_or_initialize_by(go_cardless_id: uuid)
-      local_transaction.update(
-        amount: amount,
-        currency: currency,
-        page_id: page_id,
-        subscription: subscription,
-        charge_date: charge_date,
-        customer_id: customer_id,
-        payment_method_id: payment_method_id
-      )
+      local_transaction.update amount: amount,
```

Comment:
This was one of the Do you mean: 

```
local_transaction.update(
  amount: amount,
  currency: currency
)
```
or
```
local_transaction.update({
  amount: amount,
  currency: currency
})
```
?

With the second one, :robot: complains by default but [we can configure it](https://github.com/bbatsov/rubocop/blob/master/config/default.yml#L272-L284) with

```
Style/BracesAroundHashParameters:
  EnforcedStyle: braces
```